fpage/templates/fpage/alerts.html

```diff
@@ -0,0 +1,94 @@
+<!-- LOGOUT -->
+<script>
+                    function logout(){
+                        Swal.fire({
+                            title: 'Are you sure you want to Log out?',
+                            icon: 'warning',
+                            showCancelButton: true,
+                            confirmButtonColor: '#3085d6',
+                            cancelButtonColor: '#d33',
+                            confirmButtonText: 'Yes!'
+                            }).then((result) => {
+                            if (result.value) {
+                                document.location = "/logout"
+                            }
+                        })
+                    }
+</script>
+
+<script>
+            function remove(){
```

Comment:
give proper naming like `album_remove` because if more modals are to be added there might be an error.